### ISSUE TITLE
Submit applications for virtual card providers

### ISSUE THREAD

**Issue by rndquu** [ubiquity/card-issuance#39]
Stripe has rejected our application for enabling a live mode for card issuing so we need to find a new virtual cards API provider.

There are the following virtual card providers:
- https://wallester.com/api/card-issuing
- https://karta.io/cards
- https://www.airwallex.com/core-api/issuing
- https://ufitpay.com/virtual-card
- https://www.korahq.com/
- https://union54.com/
- https://flutterwave.com/us/card-issuing
- https://www.galileo-ft.com/products/card-issuing/
- https://checkbook.io/product/payments/virtual-cards/
- https://www.modulrfinance.com/product/cards?hs_preview=vPYdytqs-105603312401
- https://www.rapyd.net/products/issuing/
- https://developer.sprint.paymentology.com/
- https://www.weavr.io/
- https://www.sagacorp.fr/en/virtual-cards-api
- https://www.berkeleypayment.com/features/virtual-cards
- https://www.adyen.com/issuing
- https://www.marqeta.com/platform/virtual-cards

I've checked those APIs and all of them support our case of issuing virtual cards for business needs.

It makes sense to reach to all of them and ask if ubiquity org is eligible to use their services.

We should make sure that (on initial contact):
1. Virtual card service provider would enable live mode for us (i.e. we're compliant with the provider's policy)
2. Virtual cards can be added to digital wallets (like Apple Wallet or Google Wallet)
3. Virtual cards can be used for payments in South Korea (or via digital wallet)

Simply put we should start applying for live mode until we get accepted somewhere.

@sergfeldman   Perhaps you could run the applications?

**Comment by sergfeldman** [comment]
@0x4007 Hey, it is not enough funds to claim rewards for this task
https://github.com/ubiquity/pay.ubq.fi/commit/bf5ca41



![1](https://github.com/ubiquity/card-issuance/assets/15567379/ce8566e1-b3df-4f1e-9f92-bae27c0b8617)

**Comment by 0x4007** [comment]
I just topped up the wallet.
